panel/template/bootstrap/bootstrap.html

```diff
@@ -29,9 +29,15 @@
     {% endif %}
     <a class="navbar-brand" href="#">
       {% if app_logo %}
-      <img src="{{ app_logo }}" height="30" class="app-logo"></img>
+      <a href="/">
+        <img src="{{ app_logo }}" height="30" class="app-logo">
+      </a>
+      {% endif %}
+      {% if app_title %}
+      <span id="app-title">
+        <a class="title" href="" >{{ app_title }}</a>
+      </span>
```

Comment:
Simply because bootstrap does not Work for me. Cf. Yesterdays test report and comment above.

Reply:
I've fixed this.

Reply:
This PR is not against the latest version, you seem to forget pulling before you start making PRs 🙂

Reply:
I don't believe so. I'm quite sure I `git fetch`, `git pull holoviz master` when I start. I think its because you do work in the mean time. Otherwise there is something technically I don't understand.

Reply:
Ok. I can see you merged the bootstrap fixes yesterday. I must be doing something wrong with git. I simply don't understand it.